src/main/sensors/battery.c

```diff
@@ -84,9 +83,9 @@ static void updateBatteryVoltage(void)
         vBatFilterIsInitialised = true;
     }
 
-    #ifdef USE_ESC_TELEMETRY
-    if (batteryConfig->batteryMeterType == BATTERY_SENSOR_ESC && isEscTelemetryActive()) {
-        vbatLatest = getEscTelemetryVbat();
+    #ifdef USE_ESC_SENSOR
+    if (feature(FEATURE_ESC_SENSOR) && batteryConfig->batteryMeterType == BATTERY_SENSOR_ESC) {
```

Comment:
Done. 
Added configuration for CLI commands:

`set current_meter_type = ESC`
`set battery_meter_type = ESC`

Default is still `ADC` when feature `ESC_SENSOR` is enabled. The user has to explicitly change the current and battery meter type to ESC. 

Current meter type is already changeable via MSP, commands `MSP_(SET_)CURRENT_METER` can be used for setting and reading the current meter type (existing code).
Changing battery meter type is not yet implemented in MSP, shall I add the `MSP_(SET_)BATTERY_METER` commands or do you have another suggestion?